Measurements/Tasks.qs

```diff
@@ -126,9 +127,29 @@ namespace Quantum.Kata.Measurements {
         // ...
         return -1;
     }
-    
-    
-    // Task 1.8. |0...0⟩ state or W state ?
+
+// Task 1.8. Distinguish two superposition states given by two arrays of bit strings
+    // Inputs:
+    //      1) N qubits in |0...0⟩ state which are guaranteed to be
+    //         in one of the two superposition states described by the given bit strings.
+    //      2) two arrays of bit strings represented as bit strings containing Bool[]s.
+    // Output: 0 if qubits were in the basis state(s) described by the first superposition bit string,
+    //         1 if they were in the basis state(s) described by the second superposition bit string.
+    // Bit values false and true correspond to |0⟩ and |1⟩ states.
+    // The state of the qubits at the end of the operation does not matter.
+    // You are guaranteed that both the arrays of bit strings will have the same length
```

Comment:
This constraints seems unnecessary? It doesn't look like distinguishing superpositions of several basis states from one another should be dependent on the exact number of terms in the superpositions - I can imagine wanting to distinguish 00 from 10 + 11